api_app/analyzers_manager/migrations/0139_analyzer_config_mobsf_service.py

```diff
@@ -0,0 +1,150 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+    ReverseManyToOneDescriptor,
+    ReverseOneToOneDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "mobsf_service.MobSF_Service",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "MobSF_Service",
+    "description": "[MobSF_Service](https://github.com/MobSF/Mobile-Security-Framework-MobSF) can be used for a variety of use cases such as mobile application security, penetration testing, malware analysis, and privacy analysis.",
+    "disabled": False,
+    "soft_time_limit": 400,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "file",
+    "docker_based": False,
+    "maximum_tlp": "RED",
```

Comment:
Even if you should have control of the MOBSF Instance, it is still an external service so by definition this should be AMBER